Content.Server/StationEvents/Events/GasLeak.cs

```diff
@@ -20,12 +20,12 @@ internal sealed class GasLeak : StationEvent
         public override string Name => "GasLeak";
 
         public override string StartAnnouncement =>
-            "Attention crew, there is a gas leak on the station. We advise you to avoid the area and wear suit internals in the meantime.";
+            Loc.GetString("station-event-gas-leak-start-announcement");
```

Comment:
just a minor formatting issue, should be on the same line as the declaration like all the other translations.